src/commands/monitors/run.rs

```diff
@@ -25,13 +28,6 @@ pub fn make_command(command: Command) -> Command {
                 .default_value("production")
                 .help("Specify the environment of the monitor."),
         )
-        .arg(
-            Arg::new("allow_failure")
```

Comment:
We can no longer support this, as sending checkin events via relay has become asynchronous.